addons/stock/i18n/stock.pot

```diff
@@ -432,8 +427,8 @@ msgstr ""
 
 #. module: stock
 #: help:stock.config.settings,group_uos:0
-msgid "Allows you to sell units of a product, but invoice based on a different unit of measure.\n"
-"For instance, you can sell pieces of meat that you invoice based on their weight."
+msgid "Allows you to store units of a product, but sell and invoice based on a different unit of measure.\n"
```

Comment:
Currently people see the translation related to the old source so it's a bit misleading but if we merge that, all existing translations will be lost. Maybe better to put a comment on that string on Transifex to warn of the true meaning of the string.